client/onboarding/index.scss

```diff
@@ -0,0 +1,28 @@
+.wcpay-wizard-task__description-element.subheading {
+	font-size: 0.9rem;
+}
+
+.wcpay-onboarding-select {
+	margin-bottom: 8px;
+
+	& button.components-custom-select-control__button {
+		width: 100%;
+		font-size: 17px;
+
+		@media ( min-width: 600px ) {
+			font-size: 14px;
+		}
+	}
+
+	.components-custom-select-control__item {
+		display: block;
+	}
+
+	& .components-custom-select-control__item-icon {
+		margin-left: 0 !important;
+	}
+}
```

Comment:
You were right, it looks a bit strange wirth the tick displaced. And the `flex-direction: row-reverse` solution doesn't work because the selected `<li>` doesn't have any unique class added.

What do you think about using the `OnboardingSelectControl` for all of them? We can make the description optional and keep all of them consistent. And also add a comment about it being used while WP Components one doesn't provide a way to add a custom option component.

Reply:
Yes, I thought about this and even added an [issue](https://github.com/Automattic/woocommerce-payments/issues/4028) to track it because I think we'll need to add some additional logic to the `OnboardingSelectControl` to make it display nicely.

For now, I will adjust them to use `OnboardingSelectControl`, then address a follow up PR where I add more flexibility to how the OnboardingSelectControl works!

Reply:
Updated in 9f2a435b532fc55251b6e217fd1c959357f642cb.